node/src/main/scala/coop/rchain/node/ReplClient.scala

```diff
@@ -29,14 +29,24 @@ class GrpcReplService(host: String, port: Int) extends ReplService[Task] {
     blockingStub.run(CmdRequest(line)).output
   }
 
-  def eval(fileName: String): Task[String] = Task.delay {
-    val file = new File(fileName)
-    if (file.exists()) {
-      withResource(Source.fromFile(file)) { source =>
-        blockingStub.eval(EvalRequest(source.getLines.mkString("\n"))).output
+  def eval(fileNames: List[String]): Task[String] = Task.delay {
+    val (existing, missing) =
+      fileNames
+        .map(new File(_))
+        .partition(_.exists())
+
+    if (missing.isEmpty) {
+      val sources = existing.map(f => Source.fromFile(f))
+
+      try {
+        val content = sources.map(_.getLines().mkString("\n")).mkString("|\n")
```

Comment:
This is doubly the wrong approach.

A) We should not be merging the lines of the file. I'm certain scala has the ability to read the entire contents of a file.
B) We should send multiple RPC's

Reply:
A) As you can see on the left, this was the original implementation. I can fix it if needed

B) See my latest commit: 93b8bc4e

Reply:
for point (A) see 6ea6c06a